File: app/src/main/java/vn/unlimit/vpngate/provider/BaseProvider.java
package vn.unlimit.vpngate.provider;

/**
 * Created by hoangnd on 1/30/2018.
 */

public class BaseProvider {
    public final static String PASS_DETAIL_VPN_CONNECTION = "PASS_DETAIL_VPN_CONNECTION";
    public final static String L2TP_SERVER_TYPE = "L2TP_SERVER_TYPE";
    public final static String FROM_LOGIN = "FROM_LOGIN";

    public static class ACTION {
        public final static String ACTION_CLEAR_CACHE = "vn.unlimit.vpngate.ACTION_CLEAR_CACHE";
        public final static String ACTION_CONNECT_VPN = "vn.unlimit.vpngate.ACTION_CONNECT_VPN";
        public final static String ACTION_CHANGE_NETWORK_STATE = "android.net.conn.CONNECTIVITY_CHANGE";
    }
}

File: app/src/main/java/vn/unlimit/vpngate/provider/BaseProvider.kt
package vn.unlimit.vpngate.provider

/**
 * Created by hoangnd on 1/30/2018.
 */
object BaseProvider {
    const val PASS_DETAIL_VPN_CONNECTION: String = "PASS_DETAIL_VPN_CONNECTION"
    const val L2TP_SERVER_TYPE: String = "L2TP_SERVER_TYPE"
    const val FROM_LOGIN: String = "FROM_LOGIN"

    object ACTION {
        const val ACTION_CLEAR_CACHE: String = "vn.unlimit.vpngate.ACTION_CLEAR_CACHE"
        const val ACTION_CONNECT_VPN: String = "vn.unlimit.vpngate.ACTION_CONNECT_VPN"
        const val ACTION_CHANGE_NETWORK_STATE: String = "android.net.conn.CONNECTIVITY_CHANGE"
    }
}
